lib/active_merchant/billing/gateways/usa_epay_transaction.rb

```diff
@@ -154,6 +155,13 @@ def add_address_for_type(type, post, credit_card, address)
         post[address_key(prefix, 'phone')]    = address[:phone]     unless address[:phone].blank?
       end
 
+      def split_names(full_name)
+        names = (full_name || '').split
```

Comment:
Should we add a guard to explicitly `return [nil, nil] if names.size <= 1`? Two reasons:
- When `''` or `nil` is passed to `split_names` it returns `["", nil]`. Feel like it should return the same value for both items.
- If name is a single word, it will end up combining the address name and credit card name in the `post`. For example, `split_names('bob') => ['', 'bob']` which would then add `fname = credit_card.first_name` and `lname = address.last_name`. Should probably use _only_ the address name or _only_ the credit card name.